examples/gallery/src/tile-3d-layer.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<head>
+  <title>deck.gl TileLayer</title>
+  <script src="https://unpkg.com/deck.gl@^8.1.0/dist.min.js"></script>
+  <script src="https://unpkg.com/@loaders.gl/i3s@2.1.6/dist/dist.min.js"></script>
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.js"></script>
+</head>
+
+<body style="margin:0; width: 100vw; height: 100vh;"></body>
+
+<script type="text/javascript">
+  const INITIAL_VIEW_STATE = {
+    longitude: -120,
+    latitude: 34,
+    height: 600,
+    width: 800,
+    pitch: 45,
+    maxPitch: 85,
+    bearing: 0,
+    minZoom: 2,
+    maxZoom: 30,
+    zoom: 14.5
+  };
+
+  const {DeckGL, Tile3DLayer} = deck;
+  const {I3SLoader} = loaders;
+
+  const deckgl = new DeckGL({
+    mapboxApiAccessToken: '<mapbox-access-token>',
+    mapStyle: 'mapbox://styles/mapbox/dark-v9',
+    initialViewState: INITIAL_VIEW_STATE,
+    controller: true,
+    onViewStateChange: ({viewState}) => {
+      deckgl.setProps({viewState});
```

Comment:
This is not necessary

Reply:
I can't interact with the map if I drop this.

Reply:
Did you drop the callback completely? Not just the setProps call?

Reply:
https://github.com/visgl/deck.gl/pull/4661/files#diff-e9dd00cf8247583ed83bba6ffdc128b1R52 needs to set `initialViewState` not `viewState`